suuchi-rocksdb/src/main/scala/in/ashwanthkumar/suuchi/store/rocksdb/RocksDbStore.scala

```diff
@@ -27,4 +27,9 @@ class RocksDbStore(config: RocksDbConfiguration) extends Store with Logging {
     log.info(s"[Closing RocksDb]")
     db.close()
   }
+
+  override def remove(key: Array[Byte]): Boolean = {
+    db.remove(key)
```

Comment:
Little of the record, above I see the `put` and `get` are synchronized in RocksStore. We don't need it right?